File: app/src/main/java/com/amazon/tv/leanbacklauncher/recommendations/RecommendationsService.java
package com.amazon.tv.leanbacklauncher.recommendations;

import com.amazon.tv.leanbacklauncher.recommendations.GservicesRankerParameters.Factory;
import com.amazon.tv.tvrecommendations.service.BaseRecommendationsService;

public class RecommendationsService extends BaseRecommendationsService {
    public RecommendationsService() {
        super(false, NotificationsServiceV4.class, new Factory());
    }
}

File: app/src/main/java/com/amazon/tv/leanbacklauncher/recommendations/RecommendationsService.kt
package com.amazon.tv.leanbacklauncher.recommendations

import com.amazon.tv.tvrecommendations.service.BaseRecommendationsService

class RecommendationsService : BaseRecommendationsService(
    false,
    NotificationsServiceV4::class.java,
    GservicesRankerParameters.Factory()
)